services/venice-controller/src/main/java/com/linkedin/venice/controller/server/AdminSparkServer.java

```diff
@@ -1,108 +1,7 @@
-import static com.linkedin.venice.controllerapi.ControllerRoute.NODE_REMOVABLE;
-import static com.linkedin.venice.controllerapi.ControllerRoute.NODE_REPLICAS;
-import static com.linkedin.venice.controllerapi.ControllerRoute.NODE_REPLICAS_READINESS;
-import static com.linkedin.venice.controllerapi.ControllerRoute.OFFLINE_PUSH_INFO;
-import static com.linkedin.venice.controllerapi.ControllerRoute.PREPARE_DATA_RECOVERY;
-import static com.linkedin.venice.controllerapi.ControllerRoute.REMOVE_DERIVED_SCHEMA;
-import static com.linkedin.venice.controllerapi.ControllerRoute.REMOVE_NODE;
-import static com.linkedin.venice.controllerapi.ControllerRoute.REMOVE_STORE_FROM_GRAVEYARD;
-import static com.linkedin.venice.controllerapi.ControllerRoute.REPLICATE_META_DATA;
-import static com.linkedin.venice.controllerapi.ControllerRoute.REQUEST_TOPIC;
-import static com.linkedin.venice.controllerapi.ControllerRoute.ROLLBACK_TO_BACKUP_VERSION;
-import static com.linkedin.venice.controllerapi.ControllerRoute.ROLL_FORWARD_TO_FUTURE_VERSION;
-import static com.linkedin.venice.controllerapi.ControllerRoute.SEND_HEARTBEAT_TIMESTAMP_TO_SYSTEM_STORE;
-import static com.linkedin.venice.controllerapi.ControllerRoute.SEND_PUSH_JOB_DETAILS;
-import static com.linkedin.venice.controllerapi.ControllerRoute.SET_MIGRATION_PUSH_STRATEGY;
-import static com.linkedin.venice.controllerapi.ControllerRoute.SET_OWNER;
-import static com.linkedin.venice.controllerapi.ControllerRoute.SET_PARTITION_COUNT;
-import static com.linkedin.venice.controllerapi.ControllerRoute.SET_TOPIC_COMPACTION;
-import static com.linkedin.venice.controllerapi.ControllerRoute.SET_VERSION;
-import static com.linkedin.venice.controllerapi.ControllerRoute.SKIP_ADMIN;
-import static com.linkedin.venice.controllerapi.ControllerRoute.STORAGE_ENGINE_OVERHEAD_RATIO;
-import static com.linkedin.venice.controllerapi.ControllerRoute.STORE;
-import static com.linkedin.venice.controllerapi.ControllerRoute.STORE_MIGRATION_ALLOWED;
-import static com.linkedin.venice.controllerapi.ControllerRoute.UPDATE_ACL;
-import static com.linkedin.venice.controllerapi.ControllerRoute.UPDATE_ADMIN_TOPIC_METADATA;
-import static com.linkedin.venice.controllerapi.ControllerRoute.UPDATE_CLUSTER_CONFIG;
-import static com.linkedin.venice.controllerapi.ControllerRoute.UPDATE_KAFKA_TOPIC_LOG_COMPACTION;
-import static com.linkedin.venice.controllerapi.ControllerRoute.UPDATE_KAFKA_TOPIC_MIN_IN_SYNC_REPLICA;
-import static com.linkedin.venice.controllerapi.ControllerRoute.UPDATE_KAFKA_TOPIC_RETENTION;
-import static com.linkedin.venice.controllerapi.ControllerRoute.UPDATE_STORAGE_PERSONA;
-import static com.linkedin.venice.controllerapi.ControllerRoute.UPDATE_STORE;
-import static com.linkedin.venice.controllerapi.ControllerRoute.UPLOAD_PUSH_JOB_STATUS;
-import static com.linkedin.venice.controllerapi.ControllerRoute.WIPE_CLUSTER;
+import static com.linkedin.venice.controllerapi.ControllerRoute.*;
```

Comment:
Can you revert this change to keep the codestyle intact? Agree it is pain to handle the disparity in codestyles between the IDE and the codebase.

However, you should be able to revert this through intellij with a click.